pkg/apis/certmanager/v1alpha2/types_certificaterequest.go

```diff
@@ -23,9 +23,16 @@ import (
 )
 
 const (
+	// Pending indicates that a CertificateRequest is still in progress.
```

Comment:
```suggestion
	// CertificateRequestReasonPending indicates that a CertificateRequest is still in progress.
```

Reply:
I don't think this is a useful change, so "resolving" this 😄